### PR TITLE
SSL certificate generation

### PR DESCRIPTION
**Pull request by an3k**

### CODE REVIEW

File: plexpy/helpers.py [hunk 1]
```diff
@@ -341,31 +341,36 @@ def split_string(mystring, splitvar=','):
 
 def create_https_certificates(ssl_cert, ssl_key):
     """
-    Create a pair of self-signed HTTPS certificares and store in them in
+    Create a self-signed HTTPS certificate and store it in
     'ssl_cert' and 'ssl_key'. Method assumes pyOpenSSL is installed.
 
-    This code is stolen from SickBeard (http://github.com/midgetspy/Sick-Beard).
+    The code were noted was stolen from SickBeard (http://github.com/midgetspy/Sick-Beard).
```
Comment (Arcanemagus) [inline plexpy/helpers.py:347]: The previous version of this sentence was much better, why did you change it?

**Comment by JonnyWong16** [comment]
#541.

File: plexpy/helpers.py (hunk 1 above)
Comment (an3k) [inline plexpy/helpers.py:347]: Most of the code wasn't even from SickBeard anymore, but you're right that my version sounded a bit bad.
